include/osireader.h

```diff
@@ -26,7 +26,7 @@ class OsiReader : public QObject, public IMessageSource
     Q_OBJECT
 
   public:
-    OsiReader(int* deltaDelay, const bool& enableSendOut, const std::string& pubPortNum, int socketType);
+    OsiReader(const int* deltaDelay, const bool& enableSendOut, std::string  pubPortNum, int socketType);
```

Comment:
Why do you work with pointers here? I think it is way better to use references? pubPostNum is not const anymore - why did you remove the reference? Only for int I think the reference does not save memory (same size pointer and int)

Reply:
clang tidy suggested me the changes :)